pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -58,6 +59,12 @@ def on_trainer_init(
         self._configure_timer_callback(max_time)
 
         # init progress bar
+        if process_position != 0:
+            rank_zero_deprecation(
+                f"Setting `Trainer(process_position={process_position})` is deprecated in v1.5 and will be removed"
+                " in v1.7. Please pass :class:`~pytorch_lightning.callbacks.progress.ProgressBar` with"
+                " ``process_position`` directly to the Trainer's `callbacks` argument instead."
```

Comment:
Can also drop the double backticks here as it won't be formatted
```suggestion
                " `process_position` directly to the Trainer's `callbacks` argument instead."
```